static/app/views/performance/newTraceDetails/traceModels/traceTree.tsx

```diff
@@ -930,10 +945,13 @@ export class TraceTree extends TraceTreeEventDispatcher {
         continue;
       }
 
+      const spanId = isEAPSpanNode(head) ? head.value.event_id : head.value.span_id;
       const autoGroupedNode = new ParentAutogroupNode(
         node.parent,
         {
-          ...head.value,
+          span_id: spanId,
+          op: head.value.op ?? tail.value.op ?? '',
+          description: head.value.description ?? tail.value.description,
```

Comment:
The old value spreads `head.value`, do we need to use `tail.value.X` here as a fallback or should we just remove those?

Reply:
Yes, I'll remove those two, thanks!